src/transaction.ts

```diff
@@ -253,6 +255,10 @@ export class Transaction implements TransactionStorageStructure {
       throw Error('firstRound must be a positive number');
     if (!Number.isSafeInteger(txn.lastRound) || txn.lastRound < 0)
       throw Error('lastRound must be a positive number');
+    if (txn.extraPages < 0 || txn.extraPages > 3)
+      throw Error(
+        'extra-pages must be an Integer between and including 0 to 3'
```

Comment:
all numbers in javascript are floats, should I manually check if the value is within {0,1,2,3}? or is this okay

Reply:
Perhaps use `Number.isInteger()` method?

Reply:
If you add a `Number.isInteger` check in addition to this, that should be sufficient.

Reply:
Ok I'll use that LOL

Reply:
before it was giving me an error but it works now that the test file is fixed